domains/certificates/docker-test-env/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.8"
+services:
+  martendb:
+    image: sibedge/postgres-plv8
+    container_name: marten
+    restart: unless-stopped
+    ports:
+      - "54320:5432"
+    volumes:
+      - ./postgres-init-data/001-plv8.sql:/docker-entrypoint-initdb.d/001-plv8.sql
```

Comment:
The file ./postgres-init-data/001-plv8.sql is empty. So what will this actually do?